actions/index.js

```diff
@@ -122,7 +122,7 @@ export const closeStandup = async (standup) => {
     for (let i = 0; i < updates.length; i++) {
       const update = updates[i];
       const message = await formatUpdateMessage(update);
-      out.push(`${u(update.user.userId)}: ${message}`);
+      out.push(`${update.user.userId}: ${message}`);
```

Comment:
You should be able to use `client` to get the slack user object, then access `display_name` from their profile:

```suggestion
      const slackUser = await client.getUser(update.user.userId);
      out.push(`${slackUser.profile.display_name}: ${message}`);
```

Reply:
(👋 @chelsea-youmans)